sunpy/net/tests/test_remote_urls.py

```diff
@@ -0,0 +1,75 @@
+import json
```

Comment:
What are these tests for now?

Reply:
these are for pinging the urls;
unfortunately, they cannot be within the plugin and have to be inside sunpy.

Reply:
fixed now,
moved them into the plugin